README.md

```diff
@@ -8,6 +8,13 @@ Extraction support functions
 
 ## Function Overview
 
+### dump_files
```

Comment:
we should consider using docstrings and generating the documentation automatically (e.g. with [sphinx](http://www.sphinx-doc.org/en/master/))